File: src/main/java/com/fish/entity/pojo/ExamTopic.java
package com.fish.entity.pojo;

import com.mybatisflex.annotation.Id;
import com.mybatisflex.annotation.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

/**
 *  实体类。
 *
 * @author fish
 * @since 2023-11-14
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(value = "oj_exam_topic")
public class ExamTopic implements Serializable {

    @Id
    private Long examId;

    @Id
    private Long topicId;

    private Long examScore;

}

File: src/main/kotlin/com/fish/entity/pojo/ExamTopic.kt
package com.fish.entity.pojo

import com.mybatisflex.annotation.Id
import com.mybatisflex.annotation.Table
import java.io.Serializable

/**
 * 实体类。
 *
 * @author fish
 * @since 2023-11-14
 */
@Table(value = "oj_exam_topic")
open class ExamTopic(
    examId: Long?,
    topicId: Long?,
    examScore: Long?
) : Serializable {
    @Id
    var examId: Long? = null
    @Id
    var topicId: Long? = null
    var examScore: Long? = null
}
